README.mkd

```diff
@@ -217,6 +217,7 @@ have the following signatures:
 
 * `void __riscv_save_<N>(void)`
 * `void __riscv_restore_<N>(void)`
+* `void __riscv_restore_tailcall_<N>(void *tail)` (LLVM/compiler-rt only)
```

Comment:
I'm not sure if putting the `tail` here as a regular-looking parameter makes it seem like this is just a normal function call where the tail is going to be received in `a0`, instead of the actual `t1`. Maybe I'm overthinking this. Is there a precedent for this or a way to make this not be an issue?

Reply:
I think I agree, would it be better for me to just write these as plain symbol names so a glance doesn't mislead people?

Reply:
I'm not sure what the best solution is. You could also add a `//` comment there or rename `tail` to `tail_in_t1` or something like that. Or just leave it as is. libgcc does seem to use C-like function prototypes (sometimes loosely), so I guess there's precedent for that, despite the corner case ABI issues. I leave this to your best judgment. If anyone else has strong feelings about this please speak up.